### PR TITLE
Update puppet for .NET auto discovery

### PR DESCRIPTION
**Pull request by jeffreyc-splunk**
**Requirements**

- [X] Content follows Splunk guidelines for style and formatting.
- [X] You are contributing original content.

**Describe the change**

For https://github.com/signalfx/splunk-otel-collector/pull/4791

### CODE REVIEW

File: gdi/opentelemetry/automatic-discovery/linux/linux-backend.rst [hunk 1]
```diff
@@ -287,15 +287,15 @@ Using the installer script, you can install and activate automatic discovery for
...
-        .. note:: By default, automatic discovery is activated for all languages (Java, Node.js, and .NET) when using the installer script. To deactivate automatic discovery for other languages, add the ``--without-instrumentation-sdk [language]`` option in the installer script command.
-
         .. tabs::
 
             .. tab:: Installer script 
 
-                .. tabs:: 
+                Using the installer script, you can install and activate automatic discovery for .NET for either all supported .NET applications on the host via the system-wide method or for only .NET applications running as ``systemd`` services.
+
+                .. note:: By default, automatic discovery is activated for all languages (Java, Node.js, and .NET) when using the installer script. To deactivate automatic discovery for other languages, add the ``--without-instrumentation-sdk [language]`` option in the installer script command.
```
Comment (aurbiztondo-splunk) [inline gdi/opentelemetry/automatic-discovery/linux/linux-backend.rst:296]: @jeffreyc-splunk - should this be 

By default, automatic discovery is activated for all languages (Java, Node.js, and .NET) when using the installer script. To deactivate automatic discovery for one of the languages, add the ``--without-instrumentation-sdk [language]`` option in the installer script command.

Comment (jeffreyc-splunk) [reply gdi/opentelemetry/automatic-discovery/linux/linux-backend.rst:296]: This was just moved to under the `.NET/nstaller script` tab without any content changes, and looks like this note is the same for Java and Node.js.

Technically, the `--without-instrumentation-sdk` option supports one or more languages, e.g. `--without-instrumentation-sdk java,nodejs` to opt-out of Java and Node.js.  Should all of these notes in this file be updated to clarify this option and supported values?

Comment (jeffreyc-splunk) [reply gdi/opentelemetry/automatic-discovery/linux/linux-backend.rst:296]: Alternatively, the `--with-instrumentation-sdk [language]` option would activate only the specified language and deactivate all other languages.

Comment (aurbiztondo-splunk) [reply gdi/opentelemetry/automatic-discovery/linux/linux-backend.rst:296]: What I meant is the original note makes it seem like there's more languages than Java/Node/Net. It's just a wording issue. I'll merge and think about it. Thanks so much for your help! :)

**Pull request merged**